plugins/GSdx/GSState.cpp

```diff
@@ -46,7 +46,7 @@ GSState::GSState()
 	, m_frameskip(0)
 	, m_crcinited(false)
 {
-	m_nativeres = !!theApp.GetConfig("nativeres", 1);
+	m_nativeres = !!theApp.GetConfig("upscale_multiplier",1)==1 ? 1:0;
```

Comment:
it's a bool variable , so I used !! just for the sort of clean code looks which ref suggested before. ofcourse it's not needed since the variable will be forced to 1 || 0